tools/python/aura_snapshot_voting/vote.py

```diff
@@ -128,14 +128,14 @@ def create_voting_dirs_for_year(base_path, year, week):
     choices = prop["choices"]
     gauge_labels = fetch_json_from_url(GAUGE_MAPPING_URL)
     gauge_labels = {
-        Web3.to_checksum_address(x["address"]): x["label"] for x in gauge_labels
+        Web3.toChecksumAddress(x["address"]): x["label"] for x in gauge_labels
```

Comment:
it might be better to use `from bal_addresses import to_checksum_address`

ref: https://github.com/BalancerMaxis/bal_addresses/blob/main/bal_addresses/utils.py

Reply:
good idea, switched in b377efc